ui/src/views/AutogenView.vue

```diff
@@ -858,6 +858,11 @@ export default {
         delete params.showunique
       }
 
+      if (['listVirtualMachinesMetrics'].includes(this.apiName) && this.dataView) {
+        delete params.state
+        delete params.details
+      }
```

Comment:
@rohityadavcloud 
can you explain why state and details are removed ?

all others look good

Reply:
These are added by the router config, I'll ping you on that code; so we must remove if the keys exists for dataView (or resource view, when only one item is being listed such as the VM)